src/main/java/com/sforce/ws/codegen/Generator.java

```diff
@@ -227,12 +226,7 @@ protected TypeMetadataConstructor newTypeMetadataConstructor(Types types, Schema
 
     protected void generateClassFromSimpleType(Schema schema, SimpleType simpleType, File dir) throws IOException {
         SimpleClassMetadata gen = new SimpleClassMetadata(schema, simpleType, typeMapper);
-        ST template;
-        if (gen.getClassName().equals("OrganizationValue")) {
-            template = templates.getInstanceOf(SIMPLE_TYPE_ORGANIZATION_VALUE);
-        } else {
-            template = templates.getInstanceOf(SIMPLE_TYPE);
-        }
+        ST template = templates.getInstanceOf(SIMPLE_TYPE);
 
         javaFiles.add(generate(gen.getPackageName(), gen.getClassName() + ".java", gen, template, dir));
```

Comment:
Screenshot that the revert lets the things be fine :
![image](https://github.com/user-attachments/assets/c116bcfc-878e-48e6-b03a-7d00cb254639)